paddle/fluid/operators/optimizers/merged_momentum_op.h

```diff
@@ -147,20 +148,45 @@ class MergedMomentumOpKernel : public framework::OpKernel<T> {
       master_params_out.clear();
     }
 
-    auto lr = ctx.Input<framework::Tensor>("LearningRate");
     auto mu = ctx.Attr<float>("mu");
     auto rescale_grad = ctx.Attr<float>("rescale_grad");
+    auto lrs = ctx.MultiInput<framework::Tensor>("LearningRate");
+    if (lrs.size() != 1) {
+      PADDLE_ENFORCE_EQ(n, lrs.size(), platform::errors::InvalidArgument(
+                                           "Input(LearningRate) number must be "
+                                           "equal to Input(Param) number."));
+    }
+    auto use_nesterov = ctx.Attr<bool>("use_nesterov");
+    auto regularization_methods =
+        ctx.Attr<std::vector<std::string>>("regularization_method");
+    auto regularization_coeffs =
+        ctx.Attr<std::vector<float>>("regularization_coeff");
+    if (regularization_methods.size() != 0) {
+      PADDLE_ENFORCE_EQ(n, regularization_methods.size(),
+                        platform::errors::InvalidArgument(
+                            "Attr(regularization_method) number must be equal "
+                            "to Input(Param) number."));
+      PADDLE_ENFORCE_EQ(n, regularization_coeffs.size(),
+                        platform::errors::InvalidArgument(
+                            "Attr(regularization_coeff) number must be equal "
```

Comment:
The size of Attr(regularization_coeff) must be equal to the size of Input(Param), but got the size of Attr(regularization_coeff) is %d, the size of Input(Param) is %d

Reply:
Try to  make the error message helpful, same for others.

Reply:
Done, tks!